### ISSUE TITLE
Rename pDOM to PDOM

### ISSUE THREAD

**Issue by zepumph** [phetsims/a11y-research#114]
From slack, there is a consensus for PDOM that involves @emily-phet. I'll make the change unless someone gets to it first.
